src/Board.cpp

```diff
@@ -0,0 +1,155 @@
+#include "Board.h"
+#include "Pawn.h"
+#include "NullPiece.h"
+#include "Player.h"
+#include "Bishop.h"
+#include "Knight.h"
+#include "Rook.h"
+#include "Queen.h"
+#include "King.h"
+
+#include <iostream>
+
+Board::Board(Player* pl1, Player* pl2) : _lastPiece(NULL)
+{
+	int i, j;
+
+	// Rooks
+	_brd[0][0] = new Rook(pl1, 0, 0, this);
```

Comment:
creating the pieces manually and not by the string passed to the chess class
if i change the string for testing i can play with empty pieces